scripts/openpose.service

```diff
@@ -17,4 +17,4 @@ ExecStart=/bin/bash -c 'source /home/amigo/.tue/setup.bash \
         _net_output_height:=368 \
         _model_folder:=/home/amigo/openpose/models/ \
         __ns:=hero/openpose \
-        image:=/hero/head_rgbd_sensor/rgb/image_raw'
+        image:=/ignore/images/topic/because/we/want/only/rosservice/requests'
```

Comment:
@LoyVanBeek You know how hackers are identified based on the things they leave in their, this is how you will be caught :rofl: